tests/python/gpu/test_forward.py

```diff
@@ -1,61 +0,0 @@
-import os
```

Comment:
it's requires downloading data and scipy. will move the test into a following integration test.